### ISSUE TITLE
Any liquid.setting is seen as an error

### ISSUE THREAD

**Issue by jordihoven** [panoply/vscode-liquid#65]
I have some Liquid tags in my <style> in order to change colors in the Shopify Customizer.

`<style>
  #bar-container {
    position: relative;
    top: 65px;
    z-index: 5;
    width: 100%;
    background-color: {{ section.settings.announcement_bg }} ; /* Having the BG color available in the customizer */
    height: 40px;
    display: flex;
    align-items: center;
    font-size: 14px;
  }
</style>`

The `{{ section.settings.announcement_bg }}` gives the error: "property value expected".

Any way of ignoring these errors?

**Comment by panoply** [comment]
Hey @Jordakoes,

This is being caused because vscode is validating the `<style>` tag and does not understand Liquid, you can disable validation on scripts and styles if you are infusing Liquid with those languages, add these settings to your vscode workspace settings file:

```json
{
  "html.validate.scripts": false,
  "html.validate.styles": false
}
```

**Comment by Jordakoes** [comment]
Hi,

Will this not cause all scripts and styles to be ignored, eg not checked? I don’t want my code editor to ignore all errors, just the ones where I infuse Liquid in Style, HTML or Script tags. 

-
Cheers,
Jordi

**Comment by aronweston** [comment]
I'm getting the same issue with script and style tags in theme.liquid. Declaration or statement expected

**Comment by bworwa** [comment]
It's a prettydiff issue, not much that can be done here. Prettydiff liquid parsing is somewhat buggy and the project is abandoned since 2019.

Unless someone is willing to retake that project or write a liquid plugin for Prettier, I don't see this being solvable.

**Comment by haroldao** [comment]
> Hey @Jordakoes,
> 
> This is being caused because vscode is validating the `<style>` tag and does not understand Liquid, you can disable validation on scripts and styles if you are infusing Liquid with those languages, add these settings to your vscode workspace settings file:
> 
> ```json
> {
>   "html.validate.scripts": false,
>   "html.validate.styles": false
> }
> ```

and on a .liquid file ? 🧐

**Comment by james0r** [comment]
Odd cause I swear I used to not have this problem with this extension a year or so ago? Am I trippin'?

**Comment by kjlibsol** [comment]
@james0r If you are tripping, so am I. :)

Anyway - the workaround described by @haroldao worked for me as well. :)

**Comment by panoply** [comment]
🚢  Shipped [v3.0.0](https://github.com/panoply/vscode-liquid/releases/tag/v3.0.0)
